circuit/src/scaffold.ts

```diff
@@ -23,7 +20,7 @@ export abstract class AxiomBaseCircuitScaffold<T> extends BaseCircuitScaffold {
   protected halo2Lib!: Halo2LibWasm;
   protected provider: string;
   protected dataQuery: DataSubquery[];
-  protected axiom: AxiomSdkCore;
+  protected axiom: AxiomV2QueryBuilder;
   protected computeQuery: AxiomV2ComputeQuery | undefined;
```

Comment:
there is already a computeQuery in queryBuilder: redundancy? can you just use the one in queryBuilder. Otherwise it will be error-prone.